pkg/operator/ceph/cluster/cluster.go

```diff
@@ -269,7 +270,7 @@ func (c *cluster) createInitialCrushMap() error {
 	k8sutil.SetOwnerRef(c.context.Clientset, c.Namespace, &configMap.ObjectMeta, &c.ownerRef)
 
 	if !configMapExists {
-		if _, err := c.context.Clientset.CoreV1().ConfigMaps(c.Namespace).Create(configMap); err != nil {
```

Comment:
This change is unnecessary.

Reply:
See above (I'll remove this change as well for now but I'll bring it back in another PR).